.ci/containers/github-differ/generate_comment.sh

```diff
@@ -114,6 +115,7 @@ retVal=$?
 if [ $retVal -ne 0 ]; then
```

Comment:
Can $? ever return a non-zero value in this case? My understanding is that +e will cause a build failure on any non-zero exit code

Reply:
I thought that was what -e did.

Reply:
Ah, I misread the block above.